src/lib.rs

```diff
@@ -60,7 +60,7 @@ use std::{
 #[cfg(feature = "internals")]
 pub use crate::{index::Index, index::IndexBuf, segment::Segment};
 use file_set::FileSet;
-use message::{MessageBuf, MessageError, MessageSet, MessageSetMut};
+pub use message::{MessageBuf, MessageError, MessageSet, MessageSetMut};
```

Comment:
Was this made `pub` to fix the example? Since `message` module is already public, I wonder if it makes sense to update the imports there instead.